src/doc/examples/ol-routing.js

```diff
@@ -0,0 +1,53 @@
+import View from 'ol/View';
+import { Map, MapboxLayer } from '../../ol';
+import RoutingControl from '../../ol/controls/RoutingControl';
+import 'ol/ol.css';
+
+export default () => {
+  const control = new RoutingControl({
+    element: document.createElement('div'),
+    apiKey: `${window.apiKey}`,
+  });
+
+  const mapboxLayer = new MapboxLayer({
+    url: `https://maps.geops.io/styles/base_bright_v2/style.json?key=${window.apiKey}`,
+  });
+
+  // eslint-disable-next-line no-unused-vars
+  const map = new Map({
+    target: 'map',
+    layers: [mapboxLayer],
+    view: new View({
+      center: [950690.34, 6003962.67],
+      zoom: 15,
+    }),
+  });
+
+  map.addControl(control);
+
+  control.activate();
```

Comment:
you shouldn't have to do this. when you call addControl , the control should call activate() automatically.